src/controllers/todo-item.controller.ts

```diff
@@ -0,0 +1,57 @@
+import { repository, EntityCrudRepository } from '@loopback/repository';
+import { TodoItem, TodoItemSchema } from '../models/index';
+import { get, put, param, patch, post, del } from '@loopback/rest';
+
+export class TodoItemController {
+  @repository('todo-item') todoItemRepo: EntityCrudRepository<TodoItem, number>;
+
+  @get('/todo/{id}/items')
+  async getTodoItems(id: number) {
+    return await this.todoItemRepo.find({
+      where: {
+        todoId: id
+      }
+    });
+  }
+
+  @post('/todo/{id}/items')
+  @param.path.number('id')
+  @param.body('todoItem', TodoItemSchema)
+  async createTodoItem(id: number, todoItem: TodoItem) {
+    todoItem.todoId = id;
+    return await this.todoItemRepo.create(todoItem);
+  }
+
+  @put('/todo/{x}/items/{id}')
+  @put('/todoitem/{id}')
+  @param.path.number('id')
+  @param.body('todoItem', TodoItemSchema)
```

Comment:
`@param.body('item', TodoItemSchema)` here or to keep it consistent as `createTodoItem`, `todoItem: TodoItem` on https://github.com/strongloop/loopback4-example-getting-started/pull/1/files#diff-9e5caab5921417309c7795160475b934R30